.github/workflows/api.yaml

```diff
@@ -1,6 +1,10 @@
 name: api
 
-on: push
+on:
```

Comment:
I added this so that all my readme changes wouldn't unnecessarily set off the CI.

Reply:
Also the CI icon is still yellow but just ignore it it isn't necessary for this PR.